privacyidea/api/lib/prepolicy.py

```diff
@@ -1941,3 +1941,34 @@ def _attestation_certificate_allowed(attestation_cert, allowed_certs_pols):
         else None
 
     return attestation_certificate_allowed(cert_info, allowed_certs_pols)
+
+
+def required_piv_attestation(request, action=None):
+    """
+    This is a token specific decorator certificate tokens for the endpoint
+    /token/init
+    According to the policy scope=SCOPE.ENROLL,
+    action=REQUIRE_ATTESTATION an exception is raised, if no attestation parameter is given.
+
+    It also checks the policy if the attestation should be verified and sets the
+    parameter verify_attestation accordingly.
+
+    :param request:
+    :param action:
+    :return:
+    """
+    from privacyidea.lib.tokens.certificatetoken import ACTION, REQUIRE_ACTIONS
```

Comment:
Could we do the import in the `if` block? that way we only do the import if the token is really a certificate token.

Reply:
Well, the philosophy about python imports.
Usually I would probably have put this at the beginning of the file.
But this actually is not that elegant, since it is a token specific import and overwirtes the ACTION class.

So I think I simply put it at the beginning of s.th.

An import in the middle of the code looks strange. 
Isn't the import no only done once, anyways?